packages/compiler/core/cli.ts

```diff
@@ -260,11 +260,7 @@ function logDiagnosticCount(diagnostics: readonly Diagnostic[]) {
   const warningCount = diagnostics.filter((x) => x.severity === "warning").length;
 
   const addSuffix = (count: number, suffix: string) =>
-    count > 1
-      ? `${errorCount} ${suffix}s`
-      : errorCount === 1
-      ? `${errorCount} ${suffix}`
-      : undefined;
+    count > 1 ? `${count} ${suffix}s` : count === 1 ? `${count} ${suffix}` : undefined;
```

Comment:
Just found out the issue for that, this already fixes it. It was because `count === 1` was `errrorCount === 1` so it would return `undefined` if there was a single warning and that would get omittted from the print.